docs/05_community-developer-tools/index.md

```diff
@@ -20,7 +19,12 @@ If you would like to submit a tool to be added to the list, please fill in [this
 
 ### Non-Fungible Token (NFT) Standards
 * [dGoods](https://github.com/MythicalGames/dgoods): An open source and free standard for handling the virtual representation of items, both digital and physical.
-* [SimpleAssets](https://github.com/EOSIO/eosjs-keygen): A simple standard for digital assets on EOSIO blockchains: Non-Fungible Tokens (NFTs), Fungible Tokens (FTs), and Non-Transferable Tokens (NTTs).
+
+* [SimpleAssets](https://github.com/CryptoLions/SimpleAssets): A simple standard for digital assets on EOSIO blockchains: Non-Fungible Tokens (NFTs), Fungible Tokens (FTs), and Non-Transferable Tokens (NTTs).
+
+* [AtomicAssets](https://github.com/pinknetworkx/atomicassets-contract): AtomicAssets is a Non Fungible Token (NFT) standard for eosio blockchains developed by [pink.network](https://pink.network/). It focuses heavily on RAM efficiency and usability.
```

Comment:
We are using the upper case of `EOSIO` in our docs. So for consistency, we can change `eosio` to `**EOSIO**`.